apps/reference/docs/guides/realtime/broadcast-presence.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: broadcast
+title: Broadcast
```

Comment:
Can we call this "Supabase Broadcast" for SEO? Can add `sidebar_label: Broadcast` to keep the sidebar label as is.

```suggestion
title: Supabase Broadcast
```